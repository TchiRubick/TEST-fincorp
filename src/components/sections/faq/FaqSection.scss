@import '../../style.scss';

.section-faq {
  width: 60%;
  margin: 2rem auto;
}