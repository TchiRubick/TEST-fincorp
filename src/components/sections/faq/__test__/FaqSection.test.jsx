import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import FaqSection from '../FaqSection';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<FaqSection />, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<FaqSection />).toJSON();
  expect(tree).toMatchSnapshot();
});
