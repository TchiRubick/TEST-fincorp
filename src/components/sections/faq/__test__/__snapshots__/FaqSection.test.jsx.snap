// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-container"
>
  <div
    className="section-faq"
  >
    <div
      className="comp-faq-card"
    >
      <div
        className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          aria-disabled={false}
          aria-expanded={false}
          className="MuiButtonBase-root MuiAccordionSummary-root"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiAccordionSummary-content"
          >
            <span
              className="comp-faq-card__title"
            >
              What are the documents required for claiming?
            </span>
          </div>
          <div
            aria-disabled={false}
            aria-hidden={true}
            className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role={null}
            tabIndex={null}
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          className="MuiCollapse-root MuiCollapse-hidden"
          style={
            Object {
              "minHeight": "0px",
            }
          }
        >
          <div
            className="MuiCollapse-wrapper"
          >
            <div
              className="MuiCollapse-wrapperInner"
            >
              <div
                role="region"
              >
                <div
                  className="MuiAccordionDetails-root"
                >
                  <span
                    className="comp-faq-card__text"
                  >
                    A pre-authorization form is required in case of cashless claims which are to be submitted to the TPA. Other documents might also be required and you should have the knowledge of the required documents.
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          aria-disabled={false}
          aria-expanded={false}
          className="MuiButtonBase-root MuiAccordionSummary-root"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiAccordionSummary-content"
          >
            <span
              className="comp-faq-card__title"
            >
              Which are the network hospitals in your vicinity?
            </span>
          </div>
          <div
            aria-disabled={false}
            aria-hidden={true}
            className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role={null}
            tabIndex={null}
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          className="MuiCollapse-root MuiCollapse-hidden"
          style={
            Object {
              "minHeight": "0px",
            }
          }
        >
          <div
            className="MuiCollapse-wrapper"
          >
            <div
              className="MuiCollapse-wrapperInner"
            >
              <div
                role="region"
              >
                <div
                  className="MuiAccordionDetails-root"
                >
                  <span
                    className="comp-faq-card__text"
                  >
                    A pre-authorization form is required in case of cashless claims which are to be submitted to the TPA. Other documents might also be required and you should have the knowledge of the required documents.
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          aria-disabled={false}
          aria-expanded={false}
          className="MuiButtonBase-root MuiAccordionSummary-root"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiAccordionSummary-content"
          >
            <span
              className="comp-faq-card__title"
            >
              Will I get covered for my pre-existing illnesses?
            </span>
          </div>
          <div
            aria-disabled={false}
            aria-hidden={true}
            className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role={null}
            tabIndex={null}
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          className="MuiCollapse-root MuiCollapse-hidden"
          style={
            Object {
              "minHeight": "0px",
            }
          }
        >
          <div
            className="MuiCollapse-wrapper"
          >
            <div
              className="MuiCollapse-wrapperInner"
            >
              <div
                role="region"
              >
                <div
                  className="MuiAccordionDetails-root"
                >
                  <span
                    className="comp-faq-card__text"
                  >
                    A pre-authorization form is required in case of cashless claims which are to be submitted to the TPA. Other documents might also be required and you should have the knowledge of the required documents.
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiPaper-root MuiAccordion-root MuiAccordion-rounded MuiPaper-elevation1 MuiPaper-rounded"
      >
        <div
          aria-disabled={false}
          aria-expanded={false}
          className="MuiButtonBase-root MuiAccordionSummary-root"
          onBlur={[Function]}
          onClick={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          role="button"
          tabIndex={0}
        >
          <div
            className="MuiAccordionSummary-content"
          >
            <span
              className="comp-faq-card__title"
            >
              Is maternity covered in health insurance policies?
            </span>
          </div>
          <div
            aria-disabled={false}
            aria-hidden={true}
            className="MuiButtonBase-root MuiIconButton-root MuiAccordionSummary-expandIcon MuiIconButton-edgeEnd"
            onBlur={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role={null}
            tabIndex={null}
          >
            <span
              className="MuiIconButton-label"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </div>
        </div>
        <div
          className="MuiCollapse-root MuiCollapse-hidden"
          style={
            Object {
              "minHeight": "0px",
            }
          }
        >
          <div
            className="MuiCollapse-wrapper"
          >
            <div
              className="MuiCollapse-wrapperInner"
            >
              <div
                role="region"
              >
                <div
                  className="MuiAccordionDetails-root"
                >
                  <span
                    className="comp-faq-card__text"
                  >
                    A pre-authorization form is required in case of cashless claims which are to be submitted to the TPA. Other documents might also be required and you should have the knowledge of the required documents.
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
